tests/v2/validation/deleting/delete_init_node.go

```diff
@@ -0,0 +1,30 @@
+package deleting
+
+import (
+	"testing"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/clusters"
+	"github.com/rancher/shepherd/extensions/nodes"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+func deleteInitNode(t *testing.T, client *rancher.Client, clusterID string) {
+	initNode, err := nodes.GetInitNode(client, clusterID)
+	require.NoError(t, err)
+
+	err = nodes.DeleteNodeRKE2K3S(client, initNode)
+	require.NoError(t, err)
+
+	logrus.Info("Awaiting machine replacement...")
```

Comment:
This can be removed since the proceeding functions already give verbose logging output.